doc/user/layouts/shortcodes/postgres-direct/right-size-the-cluster.html

```diff
@@ -4,13 +4,13 @@
 accordingly.
 
 1. Still in a SQL client connected to Materialize, use the [`ALTER CLUSTER`](/sql/alter-cluster/)
-   command to downsize the cluster to `xsmall`:
+   command to downsize the cluster to `100cc`:
 
     ```sql
-    ALTER CLUSTER ingest_postgres SET (SIZE 'xsmall');
+    ALTER CLUSTER ingest_postgres SET (SIZE '100cc');
     ```
 
-    Behind the scenes, this command adds a new `xsmall` replica and removes the
+    Behind the scenes, this command adds a new `100cc` replica and removes the
```

Comment:
There’s a reference to small on the line below this one. And an xsmall reference a the top of the page

Reply:
Thanks, fixed!